### ISSUE TITLE
About step_loss of version2

### ISSUE THREAD

**Issue by suzhenghang** [ExponentialML/Text-To-Video-Finetuning#46]
During the training of version2, the step loss easily becomes NaN, even if the learning rate is lowered. Have you encountered this issue before?

**Comment by ExponentialML** [comment]
@suzhenghang No I haven't. What type of setup are you running on? (GPU, CPU, Python version, etc.)

**Comment by suzhenghang** [comment]
Thanks，I solved this issue by disabling xformers during training. In the previous v1 version, I had it enabled.

**Comment by Rbrq03** [comment]
I encountered this issue as well. It appears to be a problem with version2 when using fp16. Disabling mixed precision can resolve the issue. However, I'm not certain about the exact cause of the problem.
